src/sentry/models/actor.py

```diff
@@ -179,17 +207,19 @@ def resolve_many(cls, actors: Sequence["ActorTuple"]) -> Sequence[Union["Team",
         return list(filter(None, [results.get((actor.type, actor.id)) for actor in actors]))
 
 
-def handle_actor_pre_save(instance, **kwargs):
+def handle_team_post_save(instance, **kwargs):
     # we want to create an actor if we don't have one
     if not instance.actor_id:
         instance.actor_id = Actor.objects.create(
-            type=ACTOR_TYPES[type(instance).__name__.lower()]
+            type=ACTOR_TYPES[type(instance).__name__.lower()],
+            team_id=instance.id,
         ).id
+        instance.save()
 
 
-pre_save.connect(
-    handle_actor_pre_save, sender="sentry.Team", dispatch_uid="handle_actor_pre_save", weak=False
-)
-pre_save.connect(
-    handle_actor_pre_save, sender="sentry.User", dispatch_uid="handle_actor_pre_save", weak=False
```

Comment:
Is it possible that during app deploy, some `pre_save` signals will be lost because the new app will no longer have the pre-save handler? Is this something to worry about?
Otherwise LGTM!

Reply:
Great question though!